pets/cassandra/on-start.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -e
+
+# Parse out hostname, formatted as: petset_name-index
+IFS='-' read -ra ADDR <<< "$(hostname)"
+MY_ID=$(expr "1" + "${ADDR[1]}")
+CFG=/etc/cassandra/cassandra.yaml
+
+i=0
+while read -ra LINE; do
+    let i=i+1
+    if [ $i = $MY_ID ]; then
+        MY_NAME=$LINE
+    fi
+    PEERS=("${PEERS[@]}" $LINE)
+done
+
+CASSANDRA_RPC_ADDRESS='0.0.0.0'
+# TODO: This is the default, and is the number of tokens assigned to this node
+# on the ring. There's probably some optimization possible here.
+CASSANDRA_NUM_TOKENS=256
+
+CASSANDRA_CLUSTER_NAME="${ADDR[0]}"
```

Comment:
This is some that I need to fix in the example as well.  Not really how we want it fixed long term